res/css/views/elements/_Field.pcss

```diff
@@ -174,8 +174,6 @@ limitations under the License.
 }
 
 .mx_Field_tooltip {
-    margin-top: -12px;
-    margin-left: 4px;
     width: 200px;
 }
```

Comment:
Afraid I'm not familiar with this stuff. Where (else) is `.mx_Field_tooltip` used, why was this necessary in the first place, and why is it no longer necessary?

Reply:
Good questions!

`.mx_Field_tooltip` is only used for `Field`s. Means it should not break other places™

I don't know the exact reason for the margin. But it seems like they are not so great aligned [when they were introduced](https://github.com/matrix-org/matrix-react-sdk/pull/2550).

![image](https://user-images.githubusercontent.com/6216686/203060187-66d5d1be-b0d1-4374-ab3d-7e4e6226eabd.png)